app/src/main/java/blockeq/com/stellarwallet/fragments/WalletFragment.kt

```diff
@@ -74,28 +74,38 @@ class WalletFragment : BaseFragment(), OnLoadAccount, OnLoadEffects {
 
     private fun bindAdapter() {
```

Comment:
I've refactored this section to only update the data if the `recyclerViewArrayList` already exists, that way the adapter is not redrawn and recreated